src/queue/client.ts

```diff
@@ -0,0 +1,12 @@
+import { Client } from '@temporalio/client';
+import { Connection as TemporalConnection } from '@temporalio/client/lib/connection';
+
+const TEMPORAL_NAMESPACE = process.env.TEMPORAL_NAMESPACE || 'default';
+
+export const getTemporalClient = async (): Promise<Client> => {
+  const connection = await TemporalConnection.connect({
+    address: 'host.docker.internal:7233',
+  });
+
+  return new Client({ connection, namespace: TEMPORAL_NAMESPACE });
```

Comment:
This will change based on how we connect to Temporal Cloud server, similar to `gatekeeper`.

Reply:
Do you maybe want only one instance so that only one connection and client are opened per app:
```ts
let temporalClient: Client;

export const getTemporalClient = async (): Promise<Client> => {
  if (!temporalClient) {
    const connection = await TemporalConnection.connect({
      address: 'host.docker.internal:7233',
    });
  
    temporalClient = new Client({ connection, namespace: TEMPORAL_NAMESPACE }) 
  }

  return temporalClient
}
```

Reply:
Yeah, we can do that too.